packages/flutter/lib/src/services/text_input.dart

```diff
@@ -472,6 +472,9 @@ TextAffinity _toTextAffinity(String affinity) {
     case 'TextAffinity.upstream':
       return TextAffinity.upstream;
   }
+  // Null affinity indicates that the platform did not provide a valid
+  // affinity. We set it to null here to allow the framework to supply
```

Comment:
Avoid "we" statements in comments.  They're confusing to non-native speakers and require extra thought to understand who "we" is referring to. (And they don't really add any information anyhow)